mediation-add-header/policy.bal

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import choreo/mediation;
+import ballerina/http;
+
+@mediation:InFlow
+public function addHeader_In(mediation:Context ctx, http:Request req, string name, string value)
+                                                                returns http:Response|false|error|() {
```

Comment:
```suggestion
                                                                returns http:Response|false|error? {
```

Reply:
Explicitly wrote `()` here to avoid any confusions/misunderstandings that it's a union of `http:Response`, `false` and `error?`